config/sets/doctrine-code-quality.php

```diff
@@ -68,5 +68,11 @@
             'time' => 'TIME_MUTABLE',
             'time_immutable' => 'TIME_IMMUTABLE',
         ]),
+        new AttributeKeyToClassConstFetch('Doctrine\ORM\Mapping\OrderBy', 'value', 'Doctrine\Common\Collections', [
```

Comment:
Right now its not fixing it, is it possible to update the value with `AttributeKeyToClassConstFetch` as the key in our fixture case is `createdAt` is generic

Reply:
@samsonasik do you know ^ Or should i create a new rule for this?

Reply:
@samsonasik sorry to poke you again i think this message split though the other messages if you can have a look then i know what the next steps are :)

Reply:
I am not sure I understand about this case, it seems issue linked is unrelated with this PR?

I guess if the usage is very specific, new rule is ok, ensure to link to correct issue id

Reply:
@samsonasik your right sorry i indeed used a wrong issue id, there is no issue for this rule. I updated the description to hopefully make it more clear: https://github.com/rectorphp/rector-doctrine/pull/166#issue-1725434453

I will create a new rule for this.

Reply:
Ready for review :)